src/main/sensors/adcinternal.h

```diff
@@ -0,0 +1,23 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+void adcInternalInit(void);
+void adcInternalProcess(void);
```

Comment:
I don't really like the name `adcInternal` for this sensor. I can't really think of something better though.

Reply:
- `mcuSensor, mcuSensorVrefint, mcuSensorTemperature (or Temp)`
- `embedded, embeddedVrefint, embeddedTemperature`
- ...

Reply:
- vrefthermo